Forwarded: https://github.com/MariaDB/server/pull/2129
Origin: https://patch-diff.githubusercontent.com/raw/MariaDB/server/pull/2129.patch
From: =?UTF-8?q?Otto=20Kek=C3=A4l=C3=A4inen?= <otto@kekalainen.net>
Date: Sun, 22 May 2022 10:13:33 -0700
Subject: [PATCH] Properly introduce wsrep_sst_backup script in project
 packaging

The script wsrep_sst_backup was introduced on MariaDB 10.3 in commit
9b2fa2a. The new script was automatically included in RPM packages but not
in Debian packages (which started to fail on waring about stray file).

Include wsrep_sst_backup in the mariadb-server-10.{3..8} package, and
also include a stub man page so that packaging of a new script is complete.

---
 debian/mariadb-server-10.6.install |  2 ++
 man/CMakeLists.txt                 |  2 +-
 man/wsrep_sst_backup.1             | 16 ++++++++++++++++
 3 files changed, 19 insertions(+), 1 deletion(-)
 create mode 100644 man/wsrep_sst_backup.1

--- a/man/CMakeLists.txt
+++ b/man/CMakeLists.txt
@@ -13,7 +13,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1335 USA
 SET(MAN1_WSREP wsrep_sst_rsync.1 wsrep_sst_common.1 wsrep_sst_mariabackup.1
-    wsrep_sst_rsync_wan.1)
+    wsrep_sst_rsync_wan.1 wsrep_sst_backup.1)
 SET(MAN1_SERVER innochecksum.1 myisam_ftdump.1 myisamchk.1
                 aria_chk.1 aria_dump_log.1 aria_ftdump.1 aria_pack.1 aria_read_log.1
                 aria_s3_copy.1
--- /dev/null
+++ b/man/wsrep_sst_backup.1
@@ -0,0 +1,16 @@
+'\" t
+.\"
+.TH "\FBWSREP_SST_BACKUP\FR" "1" "22 May 2022" "MariaDB 10\&.3" "MariaDB Database System"
+.\" -----------------------------------------------------------------
+.\" * set default formatting
+.\" -----------------------------------------------------------------
+.\" disable hyphenation
+.nh
+.\" disable justification (adjust text to left margin only)
+.ad l
+.SH NAME
+wsrep_sst_backup \- backup helper script for the MariaDB Galera Cluster
+.SH DESCRIPTION
+Use: See source code of script\.
+.PP
+For more information, please refer to the MariaDB Knowledge Base, available online at https://mariadb.com/kb/
